.github/workflows/build-distribute.yml

```
@@ -0,0 +1,144 @@
+            echo "Version name could not be extracted. Exiting." exit 1 
+          else 
+            echo "Version name is $VERSION_NAME" 
+          fi
+
+      - name: Decode keystore
+        run: |
+          echo "${{ secrets.KEYSTORE_FILE }}" | base64 --decode > ${{ github.workspace }}/keystore.jks
+          cat ${{ github.workspace }}/keystore.jks
+      - name: Build and Distribute app to Firebase
+        id: build-and-distribute
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2155:warning:12:8: Declare and assign separately to avoid masking return values

(shellcheck)